### PR TITLE
macOS desktop tracer record correct app title

### LINKED ISSUE #148: macOs windowed app tracing does not properly show Chrome apps & Electron apps
## Steps to Reproduce the Problem

1.
1.
1.

## Specifications

- Version:
- Platform:

### PR DESCRIPTION
**Pull request by rundong08**
Fixed #148

### CODE REVIEW

**Comment by rundong08** [comment]
The app name now works but the window title is still now working well. I will try to fix it.

File: pal_event_server/lib/src/loggers/app_usage/macos/apple_script_util.dart [hunk 1]
```diff
@@ -39,13 +39,13 @@ const scriptArgs = [
...
+  "-e", r'set frontAppId to (id of first application process whose frontmost is true)',
+  "-e", r'set frontAppName to title of application process id frontAppId',
   "-e",
-  r'set axwindows to (every UI element of frontApp whose role is "AXWindow" or subrole is "AXStandardWindow")',
+  r'set axwindows to (every UI element of application process id frontAppId whose role is "AXWindow" or subrole is "AXStandardWindow")',
   "-e", r'if axwindows is not {} then',
   "-e", r'set windowTitle to name of (item 1 of axwindows)',
-  "-e", r'if frontAppName is "Google Chrome" then',
+  "-e", r'if frontAppName is "Chrome" then',
   "-e", r'tell application "Google Chrome"',
```
Comment (BobEvans) [inline pal_event_server/lib/src/loggers/app_usage/macos/apple_script_util.dart:49]: Will this work if it matches frontAppName to "Chrome"?

**Comment by BobEvans** [comment]
Is this just correct title for Chrome apps? Will Electron (VS Code)  allowlist will be a separate CL?

File: pal_event_server/lib/src/loggers/app_usage/macos/apple_script_util.dart (hunk 1 above)
Comment (rundong08) [reply pal_event_server/lib/src/loggers/app_usage/macos/apple_script_util.dart:49]: Yes, it works.

**Comment by rundong08** [comment]
> Is this just correct title for Chrome apps? Will Electron (VS Code) allowlist will be a separate CL?

Electron Apps will also be correctly recognized, e.g. VS Code will be identified as `Code`.

**Comment by BobEvans** [comment]
Cool. Let me know when it is ready.

**Comment by rundong08** [comment]
Hi Bob,
This PR is now ready for review. 
Window title is now properly handled. In applescript using `of me` after an object specifier will go back to the top level "namespace".

**Comment by BobEvans** [comment]
Thanks!

**Pull request merged**